### PR TITLE
Add docs for job-log-path

### PR DESCRIPTION
**Pull request by chrisdothtml**
Adding docs for the feature added in https://github.com/buildkite/agent/pull/2174

### CODE REVIEW

File: data/content/agent_attributes.yaml [hunk 1]
```diff
@@ -127,6 +127,12 @@ attributes:
   required: false
   desc: |
       The directory to use for agent hooks. See [Hook locations](https://buildkite.com/docs/agent/v3/hooks#hook-locations) for details.
+- name: job-log-path
+  env_var: BUILDKITE_JOB_LOG_PATH
+  default_value: "platform's TempDir; e.g. '/tmp'"
+  required: false
+  desc: |
+      The directory of the temporary job log file created by the `enable-job-log-tmpfile` [agent configuration option](/docs/agent/v3/configuration). Introduced in [TBD](https://github.com/buildkite/agent/releases/tag/TBD).
```
Comment (chrisdothtml) [inline data/content/agent_attributes.yaml:135]: I left the release version as TBD as afaict this hasn't yet been released. Just looking for a review on the wording for now

**Comment by mbelton-buildkite** [comment]
Thanks for being on top of this @chrisdothtml! 

@DrJosh9000 @moskyb, can I leave this with you to include in the docs update for the next agent release?

File: data/content/agent_attributes.yaml (hunk 1 above)
Comment (DrJosh9000) [reply data/content/agent_attributes.yaml:135]: TBD = v3.50.0

Comment (DrJosh9000) [reply data/content/agent_attributes.yaml:135]: The wording here looks good though!

**Comment by chrisdothtml** [comment]
Ooh, I got confused by the GH UI, but looks like there's now a commit on the main branch with this change now. Guess we can close this PR

**Pull request closed without merging**